### DISCUSSION TITLE
Fixing Date Selection Behavior in ngx-bootstrap Datepicker After Clearing Input Box

### DISCUSSION THREAD

**Discussion by SoharabShaikh2** [valor-software/ngx-bootstrap#6716] [Q&A]
Issue in ngx-bootstrap Datepicker:
When the date is removed from the input box, selecting a new date from the calendar behaves incorrectly. Specifically:

If any text remains in the input box after clearing the date, clicking on the calendar and selecting a date causes the input box to lose focus or behave as if it was clicked outside.
The selected date does not appear in the input box on the first attempt.
The user has to open the calendar again and select the date a second time for it to display in the input box.
Expected Behavior:
Once the date is removed from the input box, selecting any new date from the calendar should immediately reflect in the input box without requiring the calendar to be reopened.

**Comment by coliff** [comment]
Looks like there are a lot of open PRs for fixes for the Datepicker component!
https://github.com/valor-software/ngx-bootstrap/pulls?q=sort%3Aupdated-desc+is%3Apr+is%3Aopen+datepicker
